### PR TITLE
feat: add redis session store

### PR DESCRIPTION
**Pull request by pk2453**

### CODE REVIEW

File: packages/forms-web-app/src/app.js [hunk 1]
```diff
@@ -87,13 +89,26 @@ if (config.server.useSecureSessionCookie) {
   app.set('trust proxy', 1); // trust first proxy
 }
 
+let sessionStoreConfig = sessionConfig();
+
+if (config.featureFlag.useSessionStore) {
+  const redisClient = createClient({ url: 'redis://redis:6379' });
```
Comment (cpcundill) [inline packages/forms-web-app/src/app.js:95]: Can we make the redis URL a config param too please?

File: packages/forms-web-app/src/config.js [hunk 2]
```diff
@@ -43,5 +43,6 @@ module.exports = {
   featureFlag: {
     googleTagManager: process.env.FEATURE_FLAG_GOOGLE_TAG_MANAGER === 'true',
     usePrivateBetaV1RoutesOnly: process.env.PRIVATE_BETA_V1_ROUTES_ONLY === 'true',
+    useSessionStore: process.env.FEATURE_SESSION_STORE === 'true',
```
Comment (cpcundill) [inline packages/forms-web-app/src/config.js:46]: Might be best to rename as `useRedisSessionStore` since the implementation is (needfully) redis-specific.

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/forms-web-app/src/app.js (hunk 1 above)
Comment (pk2453) [reply packages/forms-web-app/src/app.js]: done

File: packages/forms-web-app/src/config.js (hunk 2 above)
Comment (pk2453) [reply packages/forms-web-app/src/config.js]: done

File: packages/forms-web-app/src/config.js [hunk 3]
```diff
@@ -43,5 +34,6 @@ module.exports = {
   featureFlag: {
     googleTagManager: process.env.FEATURE_FLAG_GOOGLE_TAG_MANAGER === 'true',
     usePrivateBetaV1RoutesOnly: process.env.PRIVATE_BETA_V1_ROUTES_ONLY === 'true',
+    useRedisSessionStore: process.env.FEATURE_SESSION_STORE === 'true',
```
Comment (cpcundill) [inline packages/forms-web-app/src/config.js:37]: The env var should be `FEATURE_REDIS_SESSION_STORE`

File: docker-compose.yml [hunk 4]
```diff
@@ -59,7 +64,9 @@ services:
       USE_SECURE_SESSION_COOKIES: 'false'
       PDF_SERVICE_API_URL: 'http://pdf-service-api:3000'
       HOST_URL: 'http://localhost:9004/'
+      REDIS_URL: 'redis://redis:6379'
       PRIVATE_BETA_V1_ROUTES_ONLY: 'false'
+      FEATURE_SESSION_STORE: 'true'
```
Comment (cpcundill) [inline docker-compose.yml:69]: Should be `FEATURE_REDIS_SESSION_STORE`

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/forms-web-app/src/config.js (hunk 3 above)
Comment (pk2453) [reply packages/forms-web-app/src/config.js]: done

**Review by cpcundill — APPROVED** [review]

**Pull request merged**